src/pages/contact-us.js

```diff
@@ -0,0 +1,236 @@
+      </Text>
+
+      <div className="social-media-container">
+        <div className="social-media-labels">
+          <img src={Twitter} height={42} width={42}></img>
+
+          <Text color="#561dee" size="normal">
+            @neontribe
+          </Text>
+        </div>
+      </div>
+
+      <Text size="normal" color="black" maxCharacter="55ch">
+        We share our clients work and best practice in the tech for good sector.
+      </Text>
+    </ConstrainedWidth>
+
+    <VerticalSpacing size={3} />
+    <div className="white-background">
+      <ConstrainedWidth paddingTop="3em">
+        <Container mobileFlexDirection="column">
+          <div>
+            <Text color="black" size="normal" maxCharacter="53ch">
+              We work from home as well as in the office. Please check before
+              you plan a visit.
+            </Text>
+            <VerticalSpacing size={3}></VerticalSpacing>
+            <Container justifyContent="flex-start">
+              <Text color="#561dee">
+                <address>
+                  <p>21 Colegate </p>
+                  <p> Norwich</p>
+                  <p>NR3 1BN </p>
+                </address>
+              </Text>
+
+              <Text paddingLeft="4rem" color="#561dee">
+                <address>
+                  <p>Kaleider Studios </p>
+                  <p> 45 Preston Street</p>
+                  <p>Exeter</p>
+                  <p>EX1 1DF </p>
+                </address>
+              </Text>
+            </Container>
+          </div>
+
+          <div className="answerphone-text">
+            <Text color="black" size="normal">
+              Leave a message on our answerphone if you’d like us to call you
+            </Text>
+
+            <VerticalSpacing size={2}></VerticalSpacing>
+
+            <Text color="#561dee">0845 689 0896</Text>
+          </div>
+        </Container>
+
+        <VerticalSpacing size={9}></VerticalSpacing>
+        <Text size="small" color="black">
+          <p>Neontribe Ltd</p>
+          <p>Registered in England & Wales Registration number: 06165574</p>
+          <p>Registered office: 106 Lincoln St. Norwich, Norfolk, NR2 3LB</p>
+        </Text>
+      </ConstrainedWidth>
+    </div>
+
+    <style jsx>{`
+      p {
+        line-height: 1;
+      }
+      .logo {
+        height: 1.5rem;
+        width: 100%;
+      }
+
+      .contact-header {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+      }
+
+      .subtitle {
+        max-width: 45ch;
+      }
+
+      .white-background {
+        background-color: white;
+        padding-top: 3rem;
+        padding-bottom: 3rem;
+      }
+
+      .social-media-container {
+        width: 100%;
+        display: flex;
+        justify-content: flex-end;
+        align-items: center;
+      }
+      .social-media-labels {
+        display: flex;
+        justify-content: space-between;
+        margin-right: 10vh;
+        width: 15vh;
```

Comment:
I think using vertical height to style horizontally can have some unexpected effects, especially on large monitors.